src/main/java/uk/gov/gchq/palisade/util/ResourceBuilder.java

```diff
@@ -73,7 +73,6 @@ private ResourceBuilder() {
 
     private enum Scheme {
         file,
-        hdfs

```

Comment:
I didn't have an issue with the hdfs schema, in fact I liked that it showed how we can deal with multiple schemas. It just happens to be that they are dealt with in the same way